examples/EResNN_CIFAR10.jl

```diff
@@ -1,78 +1,93 @@
-using MAT
-using Meganet
+using MAT, Meganet
 
-vars = matread("ECNN_compareCIFAR10.mat")
-Y = vars["Y0"]
-C = vars["C"]
-theta = vars["theta"]
-Z = vars["Z"]
-timeM = vars["timeMAT"]
+n = 64;
+Y_train,C_train,Y_test,C_test = getCIFAR10(n,string(pwd(),"//..//data//CIFAR10//"));
 
-miniBatchSize = 50;
+# using PyPlot
+# y = Y_train[:,50]; y = y - minimum(y); y = y./maximum(y);
+# y = reshape(y,32,32,3);
+# y[:,:,1] = y[:,:,1]';y[:,:,2] = y[:,:,2]';y[:,:,3] = y[:,:,3]';
+# figure(); imshow(y)
+
+miniBatchSize = 64;
 nImg = [32; 32]
 cin  = 3
 nc   = [16;32;64;64]
 nt   = 2*[1;1;1]
 h    = [1.;1.;1.]
 
+
+TYPE = Float32;
+
+
+# getConvKernel = (nImg,sK) -> getConvGEMMKernel(TYPE,nImg,sK);
+# getConvKernel = (nImg,sK) -> getConvFFTKernel(TYPE,nImg,sK);
+getConvKernel = (nImg,sK) -> getSparseConvKernel2D(TYPE,nImg,sK);
```

Comment:
@lruthotto Why name an anonymous function instead of defining a one line function like below?

getConvKernel(nImg, sK) = getSparseConvKernel2D(TYPE,nImg,sK)

Reply:
what you suggest will create a new method `getConvKernel` and you might not be able to re-define the method without getting a warning. What I aim to do here, I think is easier done with an anonymous function. Probably both work the same way...

Reply:
Ahh okay, I see why you did it that way.

You're right, what I had said wouldn't work the same